bedrock/mozorg/templates/mozorg/contribute/tasks/v1/build-firefox.html

```diff
@@ -1,28 +0,0 @@
-{# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Meh, even if it's being deleted soon, at this point it would take more effort to back out the changes than to just leave it. I vote for lazy :)